### PR TITLE
styling isnt done but its functionality is there

### LINKED ISSUE #6: Set up login and register card pop ups front end

### PR DESCRIPTION
**Pull request by burtonjong**
Not sure what its supposed to look like thas why its a draft PR

image

This is what it looks like rn

### CODE REVIEW

File: src/components/login/Login.tsx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+        {loading.current ? ( // processing request
+          <p className="">Loading...</p>
+        ) : null}
+
+        {errors.name && <p className="">{errors.name.message}</p>}
+        <div className="flex flex-row gap-2">
+          <input
+            className="rounded-xl p-2"
+            type="text"
+            placeholder="Full Name"
```
Comment (LeoShangTang) [inline src/components/login/Login.tsx:85]: We should do username instead of full name unlike our website-legacy

**Review by LeoShangTang — COMMENTED** [review]
I think most of it looks good

I think we should add the capitalization function in the future when pushing up the name to the database. _Doesn't need to be done now_

Might also be a better idea to call the folder "verification" and the file name "registration" rather than just login. If you're thinking about also doing the login component in this file, I think LoginRegister is also a fine name. 

Don't worry about styling for now, it can be a seperate ticket. Functionality looks great otherwise

**Review by anthonyych4n — APPROVED** [review]
bros the goat

**Comment by burtonjong** [comment]
@LeoShangTang The only reason that its commented out right now is because of we dont have the "register" function from the backend in the repository yet, so this will lead to ESLint errors

Once those are added then we can 

And yes I am planning to add functionality for this to be a Login and a Register componen as I think less abstraction is better in these cases

I will try and add this functionality in a later ticket when things progess a little more

**Pull request merged**